scripts/mariadb-service-convert

```diff
@@ -0,0 +1,63 @@
+#!/bin/bash
+#
+# Used to generate a mariadb.service file based on the curent mysql/maridb settings
+#
+# This is to assist distro maintainers in migrating to systemd service definations from
+# a user mysqld_safe settings in the my.cnf files.
+#
+# Redirect output to user directory like /etc/systemd/system/mariadb.service.d/migrated-from-my.cnf-settings.conf
+
+tz_old=$TZ
+
+. /usr/bin/mysqld_safe --simulate
+
+echo "# converted using $0"
+echo "#"
+echo
+
+echo '[Service]'
+
+echo
+
+
+if [[ ( "$user" != "root" && "$user" != "mysql" ) || "${SET_USER}" == 1 ]]; then
+  echo User=$user
+fi
+
+
+[ -n "${open_files}" ] && echo LimitNOFILE=$open_files
+[ -n "${core_file_size}" ] && echo LimitCore=$core_file_size
+[ "${niceness}" -gt 0 ] && echo Nice=$niceness
+[ "${TZ}" != "${tz_old}" ] && echo Environment=\"TZ=${TZ}\"
+
+if [ -n "$mysqld_ld_preload" ]; then
+  new_text="$mysqld_ld_preload"
+  [ -n "$LD_PRELOAD" ] && new_text="$new_text $LD_PRELOAD"
+  echo Environment=\"LD_PRELOAD=`shell_quote_string "$new_text"`\"
+fi
+
+if [ -n "$mysqld_ld_library_path" ]; then
+  new_text="$mysqld_ld_library_path"
+  [ -n "$LD_LIBRARY_PATH" ] && new_text="$new_text:$LD_LIBRARY_PATH"
+  echo Environment=\"LD_LIBRARY_PATH=`shell_quote_string "$new_text"`\"
+fi
+
+if [ $want_syslog -eq 1 ]; then
+  echo StandardError=syslog
+  echo SyslogFacility=daemon
+  echo SyslogLevel=error
+  echo SyslogLevelPrefix=${syslog_tag_mysqld}
+fi
+
+if [ "${flush_caches}" -gt 0  ]; then
+  echo ExecStartPre=sync
+  echo ExecStartPre=sysctl -q -w vm.drop_caches=3
+fi
+
+if [ "${numa_interleave}" -gt 0  ]; then
+  echo
+  echo ExecStart=numactl --interleave=all ${cmd} '${OPTIONS}'
```

Comment:
You emit ExecStart only if numa-interleave is set. Does that mean "if numa-interleave is not set use default ExecStart specified by mariadb.service"?